includes/blocks/class-sensei-course-outline-block.php

```diff
@@ -120,11 +176,11 @@ public function render_callback( $attributes ) {
 				array_map(
 					function( $block ) {
 						if ( 'module' === $block['type'] ) {
-							return $this->get_module_block_html( $block );
+							return $this->render_module_block( $block );
```

Comment:
Can we remove the whitespace at the end of line 171? My editor complains about it.

Reply:
👾